### PR TITLE
Breadcrumb integrated in translation interface

### PR DESCRIPTION
**Pull request by abhiprojectz**
**Bug:**

As per issue [#375](https://github.com/sawaliram/sawaliram/issues/375) breadcrumb is to be integrated on translation interface.

**Description:**

This fixes [#375](https://github.com/sawaliram/sawaliram/issues/375)
Breadcrumb is integrated.

### CODE REVIEW

File: dashboard/views.py [hunk 1]
```diff
@@ -1604,7 +1604,7 @@ def get_form(self):
         form = super().get_form()
 
         # Decide language options
-        available_languages = self.answer.list_available_languages()
```
Comment (badrihippo) [inline dashboard/views.py:1607]: Why is this line showing up here? It's not relevant for this PR, is it?

**Review by badrihippo — CHANGES_REQUESTED** [review]
Looking good except for one extra line that should probably not be changed in *this* PR

File: dashboard/views.py (hunk 1 above)
Comment (abhiprojectz) [reply dashboard/views.py:1607]: no its relevant to this PR as these changes are checked so, i put it up here so the testing of the PR can be done easily also for article translation too.

**Comment by abhiprojectz** [comment]
Removed those changes that were left after being **checked** as in [#377](https://github.com/sawaliram/sawaliram/pull/377) PR, for the sake of accurate checking of the PR.

**Review by badrihippo — APPROVED** [review]
Nice work! I'd suggest "rebase and merge" for this commit, but if that doesn't work, second best option is "squash and merge"

**Pull request merged**